Fenced_Frames_Ads_Reporting.md

```diff
@@ -63,7 +63,7 @@ Browser will process this similar to how the existing [navigator.sendBeacon](htt
 ```
 window.fence.reportEvent({
   'eventType': 'click',
-  'eventData': {'clickX': '123', 'clickY': '456'},
+  'eventData': JSON.stringify({'clickX': '123', 'clickY': '456'}),
```

Comment:
As discussed offline, let's add another example without a key-value pair to clarify that it's an arbitrary string.

Reply:
Updated, thanks!